zutil.c

```diff
@@ -77,9 +77,6 @@ unsigned long ZEXPORT PREFIX(zlibCompileFlags)(void)
 #ifdef ZLIB_WINAPI
     flags += 1 << 10;
 #endif
-#ifdef BUILDFIXED
-    flags += 1 << 12;
-#endif
```

Comment:
Does it make sense to put in a comment that the 1 << 12 flag was previously used by _BUILDFIXED_ or does it not matter?

Reply:
We have already removed several flags from that area already,
and they are all documented in zlib-ng.h for the zng_zlibCompileFlags function, so I don't think we need to duplicate it here.